frontend/cypress/integration/common/overview.ts

```diff
@@ -23,7 +23,7 @@ Before(() => {
 
 Given('user opens the overview page', () => {
     // Forcing "Pause" to not cause unhandled promises from the browser when cypress is testing
-    cy.visit(url + '/overview?refresh=0');
+    cy.visit(url + '/overview?refresh=0&duration=60');
```

Comment:
I'm not sure if this would trigger then the change in this other test:
![image](https://user-images.githubusercontent.com/1662329/164420496-e21ebe37-91cc-4653-8431-f35de7e9cebd.png)

But let's consolidate this change, as I also expect to update these tests with the changes in the "data-test" attributes discussed in the other WIP.